src/__fixtures__/nested-workspaces-with-scoped-package-names/packages/foo/package.json

```diff
@@ -6,6 +6,6 @@
   },
   "dependencies": {
     "react": "^15.6.1",
-    "bar": "^1.0.0"
+    "@scoped/bar": "^1.0.0"
```

Comment:
Hmm... I'm going to double check all these fixures. I dont recall changing all these, so I'm thinking my tests may have modified them at some point.

- [ ] find out if I modified all these fixtures or not...

Reply:
Yup, all good.